linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -1102,6 +1101,12 @@ private[emitter] final class ClassEmitter(jsGen: JSGen) {
       case ModuleKind.NoModule =>
         genAssignToNoModuleExportVar(exportName, exportedValue)
 
+      case ModuleKind.ESModule =>
+        val field = envField("e_" + exportName)
```

Comment:
`envField("e", exportName)`? There seems to be an overload with a `subfield` parameter.